app/main.py

```diff
@@ -14,4 +14,11 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    count = 0
+
+    for char in phrase:
+        if char.lower() == letter.lower():
+            count += 1
+
+    return count
```

Comment:
flake8 not passed, and rewrite this function in one line 
<img width="1047" alt="Screenshot 2023-08-28 at 16 46 24" src="https://github.com/mate-academy/py-count-occurrences/assets/117740672/b201b78d-6ac5-4f98-a78b-e34c6e561b5d">